experiments/golden-results/StratoX-summary.txt

```diff
@@ -2205,6 +2205,11 @@ Error detected at REDACTED
 --
 Occurs: 1 times
 +===========================GNAT BUG DETECTED==============================+
+| GNU Ada (ada2goto) Assert_Failure tree_walk.adb:5051                     |
```

Comment:
Can the "Update golden results" commit please include information for why the golden results here have changed? I'd have expected these to not change at all.

Reply:
I think these were previously *missed* by the old version of the script. There seems to be a couple of cases where gnat2goto fails with an assertion failure in the middle of outputing the GNAT BUG DETECTED report - meaning it never gets to print the ending deliminator. The new script handles this better so it actually spots these cases and reports them.